test/uk/gov/hmrc/gform/binders/ValueClassBinderSpec.scala

```diff
@@ -1,57 +0,0 @@
-/*
- * Copyright 2023 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package uk.gov.hmrc.gform.binders
-
-import org.scalatestplus.play.PlaySpec
-import uk.gov.hmrc.gform.models.FastForward
-import uk.gov.hmrc.gform.sharedmodel.formtemplate.SectionOrSummary
-import uk.gov.hmrc.gform.sharedmodel.formtemplate.SectionNumber
-
-class ValueClassBinderSpec extends PlaySpec {
```

Comment:
Why is this deleted?

Reply:
It's testing [`ValueClassBinder.fastForwardListBinder`](https://github.com/hmrc/gform-frontend/pull/2077/files#diff-20adfff1866bae740f327735cd1c114c81a1220eaa1dd882c907a326a47cb827L244). I think we don't need this implementation.